src/styles/main.css

```diff
@@ -0,0 +1,19 @@
+section {
+  display: flex;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  margin-left: 100px;
+}
+
+ul {
+  list-style-type: none;
+  border: 1px solid black;
+}
+
+li {
+  margin-left: -30px;
+}
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly adjust your styling to match the wireframe as much as you can below:


*wireframe*
![wireframe](https://user-images.githubusercontent.com/59397159/173563780-23b0a66f-759f-4685-82de-dabba6f59921.png)

*your design*
![wf](https://user-images.githubusercontent.com/59397159/173564062-e561e3ab-1290-41eb-8d3b-71eaf01d78cf.png)

- The refresh button is aligned at the right on the same line with the `Recent score`
- The list items have a background which could be applied using the CSS nth-child() selector. e.g 

` li:nth-child(even) {
  background-color: gray;
}
`
- The form submit button is aligned at the right. please kindly fix that.